web/vital_records/views/death.py

```diff
@@ -24,8 +24,24 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CountyView(common.CountyView):
-    pass
+class CountyView(ValidateTypeMixin, common.CountyView):
+    form_class = CountyForm
+    step_name = Steps.county_of_death
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["form_question"] = "What was the county of death?"
+        context["form_hint"] = (
+            "We can only issue death records for deaths that occurred in California. If the death took place in a "
+            "different state, please contact the Vital Records office in the state the death occurred in to request "
+            "a new record."
+        )
+        context["font_hint_name"] = "county-hint"
```

Comment:
I think there may be a small typo here where `font_hint_name` should be `form_hint_name` to match the name in the [template](https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/blob/main/web/vital_records/templates/vital_records/request/form.html#L8)?

Reply:
:scream:  good eyes @lalver1 ... it looks like this typo actually occurs in multiple places, and the effect of it is that the `aria-describedby` and `id` on the fieldset and element respectively won't be set, so it's definitely a user-facing bug.

I will open a separate ticket / PR to fix this.

Reply:
Oh yep, I see that covers it already :+1:

Reply:
Fixed the typo in this PR with rebased commit [7fdabe143](https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/pull/397/commits/7fdabe143a5ec6136ddb8328e5476907cfac5d04#diff-6b295ea2f122b94ee15ed2d2454b4bb341fcfa314b78f05762e079601a2f2aa4R39)

Reply:
Yes! Sorry completely forgot to mention #400 ! 😅